wp-includes/components/class-wp-service-worker-navigation-routing-component.php

```diff
@@ -473,8 +474,9 @@ public function get_priority() {
 	 * @return string Script.
 	 */
 	public function get_script() {
-		$script = file_get_contents( PWA_PLUGIN_DIR . '/wp-includes/js/service-worker-navigation-routing.js' ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
-		$script = preg_replace( '#/\*\s*global.+?\*/#', '', $script );
+		$script  = file_get_contents( PWA_PLUGIN_DIR . '/wp-includes/js/service-worker-offline-commenting.js' ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
```

Comment:
Originally it was implemented separately but then moved to here since it seemed to make sense due to sharing some logic (getting the URL-s to replace for example, `$offline_error_precache_entry`). Makes sense though that the default handler should probably have a late priority! Will move the commenting separately.

Reply:
Also getting the error message would be shared then so in case of moving it separately we could maybe still have `wp_service_worker_get_error_messages` as a global method.